configuration/dto.go

```diff
@@ -54,7 +54,6 @@ type RealmAdminConfiguration struct {
 	ConsentRequiredSocial                 *bool           `json:"consent_required,omitempty"`
 	ConsentRequiredCorporate              *bool           `json:"consent_required_corp,omitempty"`
 	ShowGlnEditing                        *bool           `json:"show_gln_editing,omitempty"`
-	BusinessIDIsNotGLN                    *bool           `json:"business_id_is_not_gln,omitempty"`
```

Comment:
If field is provided, did you checked that it does not make the json.Unmarshall fail?

Reply:
Yes, when a field is provided that does not have a match in the struct it is ignored by Unmarshall

https://pkg.go.dev/encoding/json#Unmarshal